src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -433,6 +433,12 @@ def _generate_template_progress(self, correlation_id):  # pylint: disable=no-sel
 
     def __call__(self, poller):
         from msrest.exceptions import ClientException
+        import colorama
+        import platform
+
+        # https://github.com/azure/azure-cli/issues/3555
+        if platform.system() == 'Windows':
```

Comment:
If the `colorama.init()` doesn't have a side-effect on the system other than Windows, we don't need this condition test.